catalog/index.html

```diff
@@ -0,0 +1,344 @@
+              <td class="more-goods__cell">100 р.</td>
+              <td class="more-goods__cell">
+                <button class="more-goods__btn button">Заказать</button>
+              </td>
+            </tr>
+            <tr class="more-goods__string">
+              <td class="more-goods__cell more-goods__name">Витамины</td>
+              <td class="more-goods__cell">1 упаковка (30 г)</td>
+              <td class="more-goods__cell">300 р.</td>
+              <td class="more-goods__cell">
+                <button class="more-goods__btn button">Заказать</button>
+              </td>
+            </tr>
+          </table>
+          <div class="more-goods__gift">
+            <img
+              class="more-goods__gift-pic"
+              src="../img/gift.svg"
+              width="55"
+              height="65"
+              alt="Подарок"
+            />
+            <p class="more-goods__offer">
+              Закажите все и получите чехол для кота в подарок!
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="map">
+        <div class="map__content">
+          <p class="map__cooperation">Приглашаем к сотрудничеству дилеров!</p>
+          <div class="map__text-wrap">
+            <p class="map__adress">ул. Большая Конюшенная, д. 19/8</p>
+            <p class="map__adress">Санкт-Петербург</p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="footer__wrapper container">
+        <img
+          class="footer__logo"
+          src="../img/footer-logo.svg"
+          width="128"
+          height="24"
+          alt="Логотип кошачьей энергии"
+        />
+        <ul class="footer-socials">
+          <li class="footer__item">
+            <a class="footer__link" href="#"
+              ><img
+                class="footer__vk"
+                src="../img/vk.svg"
+                width="30"
+                height="16"
+                alt="вконтакте"
+            /></a>
+          </li>
+          <li class="footer__item">
+            <a class="footer__link" href="#"
+              ><img
+                class="footer__inst"
+                src="../img/insta.svg"
+                width="23"
+                height="23"
+                alt="инстаграм"
+            /></a>
+          </li>
+          <li class="footer__item">
+            <a class="footer__link" href="#"
+              ><img
+                class="footer__fb"
+                src="../img/fb.svg"
+                width="14"
+                height="28"
+                alt="фейсбук"
+            /></a>
+          </li>
+        </ul>
+        <div class="footer__copyright copyright">
+          <p class="copyright__text">HTML Academy</p>
+          <a class="copyright__link" href="#"
+            ><img
+              class="copyright__logo"
+              src="../img/htmlacademy.svg"
+              alt="логотип академии"
```

Comment:
`alt` is better when it replaces its contents semantically: here it should be "HTML Academy", but since we have this text already on a previous line, we can omit alt text and leave the attribute empty.